pyproject.toml

```diff
@@ -11,13 +11,19 @@ uvicorn = "^0.12.2"
 detect-secrets = "^0.14.3"
 SQLAlchemy = "^1.3.20"
 GitPython = "^3.1.9"
-pydantic = {extras = ["dotenv"], version = "^1.6.1"}
+pydantic = {extras = ["typing_extensions"], version = "^1.7.3"}
```

Comment:
Be careful you lost the `dotenv` extension here. And do you need `typing_extensions`? The Pydantic docs say "use of Literal prior to python 3.8 relies on typing-extensions" and you're on 3.8+

Reply:
When I do poetry add pydantic[dotenv], it will remove typing_extensions and not append dotenv package
I added dotenv and removed typing_extensions

Reply:
Yes, if you need multiple extras you have to specify them all in one go, like `poetry add requests[security,socks]` (https://python-poetry.org/docs/cli/#add)